packages/hash/frontend/src/components/hooks/useUsers.ts

```diff
@@ -1,53 +1,52 @@
 import { useMemo } from "react";
 import { useQuery } from "@apollo/client";
-import { GetAccountsQuery } from "../../graphql/apiTypes.gen";
-import { getAccounts } from "../../graphql/queries/account.queries";
-import { useRouteAccountInfo } from "../../shared/routing";
+import { types } from "@hashintel/hash-shared/types";
+import {
+  GetAllLatestPersistedEntitiesQuery,
+  GetAllLatestPersistedEntitiesQueryVariables,
+} from "../../graphql/apiTypes.gen";
+import { getAllLatestEntitiesQuery } from "../../graphql/queries/knowledge/entity.queries";
+import { isEntityVertex, Subgraph } from "../../lib/subgraph";
+import { useInitTypeSystem } from "../../lib/use-init-type-system";
+import { constructUser, User } from "../../lib/user";
 
 export const useUsers = (): {
   loading: boolean;
-  data: {
-    entityId: string;
-    shortname: string;
-    name: string;
-    isActiveOrgMember: boolean;
-  }[];
+  users: User[];
 } => {
-  const { data, loading } = useQuery<GetAccountsQuery>(getAccounts);
-  const { accountId: workspaceAccountId } = useRouteAccountInfo();
+  const { data, loading } = useQuery<
+    GetAllLatestPersistedEntitiesQuery,
+    GetAllLatestPersistedEntitiesQueryVariables
+  >(getAllLatestEntitiesQuery, {
+    /** @todo reconsider caching. This is done for testing/demo purposes. */
```

Comment:
This request is failing because of missing variables (dataTypeResolveDepth, propertyTypeResolveDepth, linkTypeResolveDepth, entityTypeResolveDepth, linkResolveDepth, linkTargetEntityResolveDepth).